samples/msal-node-samples/auth-code/index.js

```diff
@@ -29,21 +31,51 @@ const cachePlugin = require('../cachePlugin')(cacheLocation);
 const scenario = argv.s || "customConfig";
 const config = require(`./config/${scenario}.json`);
 
+const sessionConfig = {
+    secret: 'ENTER_YOUR_SECRET_HERE',
+    resave: false,
+    saveUninitialized: false,
+    cookie: {
+        secure: false, // set this to true on production
+    }
+}
+
 // Sample Application Code
 const getTokenAuthCode = function (scenarioConfig, clientApplication, port) {
     // Set the port that the express server will listen on
     const serverPort = port || SERVER_PORT;
     // Create Express App and Routes
     const app = express();
 
+    app.use(session(sessionConfig))
+
     const requestConfig = scenarioConfig.request;
 
     app.get("/", (req, res) => {
         const { authCodeUrlParameters } = requestConfig;
+
+        const cryptoProvider = new msal.CryptoProvider()
             
         if (req.query) {
             // Check for the state parameter
             if(req.query.state) authCodeUrlParameters.state = req.query.state;
+            // Check for nonce parameter
+            /**
+             * MSAL node provides for the use of OIDC nonce feature which is used to protect against token replay.
```

Comment:
I'm nitpicking here, just a small suggestion
```suggestion
             * MSAL Node supports the OIDC nonce feature which is used to protect against token replay.
```